pkg/performanceprofile/controller/performanceprofile/components/machineconfig/machineconfig.go

```diff
@@ -60,6 +60,7 @@ const (
 	setCPUsOffline            = "set-cpus-offline"
 	setRPSMask                = "set-rps-mask"
 	clearIRQBalanceBannedCPUs = "clear-irqbalance-banned-cpus"
+	cpusetConfigure           = "cpuset-configuration"
```

Comment:
You should add the script itself to the system using [addContent](https://github.com/openshift/cluster-node-tuning-operator/blob/3dd2403b1617e612a09f9f3b9ff20a6ce0035b80/pkg/performanceprofile/controller/performanceprofile/components/machineconfig/machineconfig.go#L434)

Reply:
I see I thought that was taken care of above but I was wrong, thanks!